docs/framework/resources/packaging-and-deploying-resources-in-desktop-apps.md

```diff
@@ -122,10 +123,44 @@ Applications rely on the .NET Framework Resource Manager, represented by the <xr
   
 -   If the probe for a particular resource assembly fails, the runtime does not raise the <xref:System.AppDomain.AssemblyResolve?displayProperty=nameWithType> event.  
   
+
+### .NET Core resource fallback process
+ The .NET Core resource fallback process involves the following steps:
+
+1.  The runtime attempts to load a satellite assembly for the requested culture.
+     * Checks the directory of the currently executing assembly for a subdirectory that matches the requested culture. If it finds the subdirectory, it searches that subdirectory for a valid satellite assembly for the requested culture and loads it.
+
+       > [!NOTE]
+       >  On operating systems with case sensistive file systems (e.g. Linux and OSX) the culture name subdirectory search is case sensitive.  The subdirectory name must exactly match the case of the <xref:System.Globalization.CultureInfo.Name?displayProperty=nameWithType> i.e. `es` or `es-MX`.
```

Comment:
- case sensistive --> case-sensitive
- (e.g., Linux and OSX) --> (that is, Linux and macOS), (note the comma)
- i.e \`es\` or \`es-MX\` --> (for example, \`es\` or \`es-MX\`)

Reply:
Done